icu4c/source/data/rbnf/root.txt

```diff
@@ -661,6 +661,8 @@ root{
             "%%tamil-thousands:",
             "0: =%tamil=;",
             "1000: <<\u0BF2[>>];",
+            "%zDefault:",
+            "0: =#,##0=;",
```

Comment:
Three things: 1) I'm still not sure this is the right thing to do.  It produces a more meaningful result if you forget to set the default result set on the NumberingSystem formatter, but that might make it harder to tell what you've done wrong (or that you've done anything wrong) if you get this in your app.  2) I didn't know how to get this change into icudata.jar for use on the Java side.  3) Do we also need to make this change in CLDR, and if we don't, how will we keep it here with future integrations of data from CLDR?

Reply:
Yes, there should be a CLDR ticket for this, all RBNF data comes from CLDR. As for moving this to the ICU4J jars, the procedure for that is part of icu4c/source/data/cldr-icu-readme.txt

Reply:
Thank you.  I think I've fixed `icudata.jar` now, and I filed https://unicode-org.atlassian.net/browse/CLDR-15863 for the CLDR side of the fix.